diff --git a/dewpoint/aws.py b/dewpoint/aws.py
index 91d058f..54f794d 100644
--- a/dewpoint/aws.py
+++ b/dewpoint/aws.py
@@ -133,7 +133,7 @@ class AWSClient(object):
         req = urllib2.Request(url, data, headers)
         try:
             resp = self.opener.open(req)
-        except urllib2.HTTPError, e:
+        except urllib2.HTTPError as e:
             raise self.parse_httperror(e)
 
         i = resp.info()
